utilities/rpp-unittests/HOST_NEW/Tensor_host_pln3.cpp

```diff
@@ -913,6 +916,58 @@ int main(int argc, char **argv)
 
         break;
     }
+    case 6:
+    {
+        test_case_name = "jitter";
+
+        Rpp32u kernelSizeTensor[images];
+        Rpp32u seed = 1255459;
+        for (i = 0; i < images; i++)            
+        {
+            kernelSizeTensor[i] = 5;
+        }
+
+        // Uncomment to run test case with an xywhROI override
+        /*for (i = 0; i < images; i++)
+        {
+            roiTensorPtrSrc[i].xywhROI.xy.x = 0;
+            roiTensorPtrSrc[i].xywhROI.xy.y = 0;
+            roiTensorPtrSrc[i].xywhROI.roiWidth = 100;
+            roiTensorPtrSrc[i].xywhROI.roiHeight = 180;
+        }*/
+
+        // Uncomment to run test case with an ltrbROI override
+        /*for (i = 0; i < images; i++)
+        {
+            roiTensorPtrSrc[i].ltrbROI.lt.x = 50;
+            roiTensorPtrSrc[i].ltrbROI.lt.y = 30;
+            roiTensorPtrSrc[i].ltrbROI.rb.x = 210;
+            roiTensorPtrSrc[i].ltrbROI.rb.y = 210;
+        }
+        roiTypeSrc = RpptRoiType::LTRB;
+        roiTypeDst = RpptRoiType::LTRB;*/
+
+        start_omp = omp_get_wtime();
+        start = clock();
+        if (ip_bitDepth == 0)
```

Comment:
Please revert the changes done in old test suite files too

Reply:
Done